test/functionalTest/cases/117_convop_using_standard_ops/postContextEntitiesByEntityId.test

```diff
@@ -0,0 +1,332 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+ConvOp postContextEntitiesByEntityId: POST /v1/registry/contextEntities/{entityId::id}
+
+--SHELL-INIT--
+dbInit CB
+dbInit CP1
+brokerStart CB
+brokerStart CP1
+
+--SHELL--
+
+#
+# 01. Register E1 using POST /v1/registry/contextEntities/E1
+# 02. Discover E1
+# 03. Update the registration of E1, CP1 as providing application, using postEntityByIdAttributeByName
+# 04. Discover E1
+# 05. Create E1/A1 in CP1
```

Comment:
So you are mixing in this .test the CPr functionaly with "basic NGSI9" usage (register/discover).

I undertand this is intentional but... maybe we are mixing too many things in the same place?